modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,17 +1,65 @@
 // Module included in the following assemblies:
 //
-// virt/install/virt-installing-virtctl.adoc
+// virt/install/virt-enabling-virtctl.adoc
 
 [id="virt-installing-virtctl-client_{context}"]
 = Installing the virtctl client
 
-Install the `virtctl` client from the `kubevirt-virtctl` package.
+Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+
+.Prerequisites
+
+* You must have downloaded the `virtctl` client.
 
 .Procedure
 
-* Install the `kubevirt-virtctl` package:
+* For Linux:
+
+. Extract the tarball. The following CLI command extracts it into the same directory as the tarball and is applicable for all distributions:
++
+[source,terminal]
+----
+$ tar -xvf <virtctl-version-distribution.arch>.tar.gz
+----
+
+. Navigate the extracted folder hierachy and make the `virtctl` binary executable:
```

Comment:
From the [contribution guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output):

```
If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:

<Step description>, run the following command:

<Step description>, enter the following command:
```
Suggested change: "Navigate the extracted folder hierachy and run the following command to make the `virtctl` binary executable:"